Tests/Tracking/Util/StringAnyEncodable.swift

```diff
@@ -19,9 +19,73 @@ struct DummyData: Codable, Equatable {
     }
 }
 
+struct Unencodable {
+    let data: Any
+}
+
 class StringAnyEncodableTest: UnitTest {
+    
+    func test_stringanyencodable_handles_unencodable_data() {
+        let expect = #"{}"#
 
-    func test_stringanyencodable_encodes_properly() {
+        let data = ["foo": Unencodable(data: 12345)] as [String:Any]
+        
+        let json = StringAnyEncodable(data)
+        
+        guard let actual = jsonAdapter.toJson(json, encoder: nil) else {
+            XCTFail("couldn't encode to JSON")
+            return
+        }
+        
+        XCTAssertEqual(expect, String(decoding:actual, as:UTF8.self))
```

Comment:
To make this a little less copy/paste, the SDK does have [an extension](https://github.com/customerio/customerio-ios/blob/664819caf48d23168ce97e027e51f8040b7922d1/Sources/Tracking/Extensions/DataExtensions.swift#L4) you can use here. 

```swift
actual.string
```